File: tournaments/divisorsSubset/divisorsSubset.js
function divisorsSubset(subset, n) {

    var res = 0;
  
    for (var i = 1; i <= n; i++) {
        var correct = true;
        for (var j = 0; j < subset.length; j++) {
            if (i % subset[j] !== 0) {
            correct = false;
            }
        }
        if (correct) {
            res++;
        }
    }
  
    return res;
}
    

File: tournaments/divisorsSubset/divisorsSubset.cpp
int divisorsSubset(std::vector<int> subset, int n) {

  int res = 0;

  for (int i = 1; i <= n; i++) {
    bool correct = true;
    for (int j = 0; j < subset.size(); j++) {
      if (i % subset[j] != 0) {
        correct = false;
      }
    }
    if (correct) {
      res++;
    }
  }

  return res;
}
